commands/plugin.go

```diff
@@ -107,6 +107,7 @@ func pluginAddCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 		if err != nil {
 			printer.PrintError("Unable to add plugin: " + args[i] + ". Error: " + err.Error())
+			return -1, errors.New("Unable to add plugin:  " + plugin )
```

Comment:
The function does not return multiple values, you just need to accumulate the error with `multiError.Error` and return it in the end. And also update the tests and so on.

Reply:
We're using https://pkg.go.dev/github.com/hashicorp/go-multierror@v1.1.1 in `mmctl`.
You can use the `Append` function to collect the errors.

Reply:
What about changing the return to (int, error)?  
`func pluginAddCmdF(c client.Client, cmd *cobra.Command, args []string) (int, error) {`